joomla/joomla-cms
# [5.2] Small fix for the dt- tag in content details list

Pull Request for Issue # .

### Summary of Changes
The article info in articles is built as a definition list.
The definion term is the text for Details,
The definition description is a list of different items like author, publishing-date ad so on

If the user decides to hide the definition term then the dt-tag remains empty.
With this small PR the the details info is not empty but hides the text only for sighted users, screen readers still have the complete definition term. 

This PR is of type "better than nothing" 
because a more correct solution for the would be a definition list which writes for every element 
`<dt>Author</dt><dd>Mr. John Doe</dd>`


### Testing Instructions
Easiest test is any blog in joomla. 
Just set some detail info in the corresponding menu item to "show" 
Then show and hide the details 
No change in frontend for sighted users, but check the source code.

### Actual result BEFORE applying this Pull Request
If the Article Info Title is set on "hide". The dt is empty 
![grafik](https://github.com/joomla/joomla-cms/assets/1035262/1bf22b2f-c248-4900-8e61-17fb7d07acff)


### Expected result AFTER applying this Pull Request
![grafik](https://github.com/joomla/joomla-cms/assets/1035262/77fc90f0-c752-41a1-b1bd-f1d06d439efe)



### Link to documentations
Please select:
